vfs/vfs.go

```diff
@@ -13,9 +13,29 @@ import (
 	"time"
 
 	"github.com/cozy/cozy-stack/couchdb"
+	"github.com/cozy/cozy-stack/couchdb/mango"
 	"github.com/spf13/afero"
 )
 
+// Indexes is the list of required indexes by the VFS inside CouchDB.
+var Indexes = []mango.Index{
+	mango.Index{
+		Name:  "FileFromParent",
```

Comment:
Name And DDoc are couchdb internals : "_design/{Ddoc}" is the name of the design document where the index  is defined. "name" is the name of this index in the design doc. 

If we dont let couchdb pick those automatically, there will be a wasteful double indexing if two apps define indexes on same field but with different name.